### ISSUE TITLE
Feature Request: Support both JDK 8 and JDK 11

### ISSUE THREAD

**Issue by wyyang** [sleroy/sonar-slack-notifier-plugin#5]
In many orgs both JDK 8 and JDK 11 are used at different teams. The current version of the plug in stops people using JDK 8 from using SonarQube to parse their code. It will be best if both are supported. Thanks!

**Comment by sleroy** [comment]
I agree with you but at this time, I'm alone to maintain this plugin and
with my limited time, I cannot maintain several versions at the same time.

I have as many requests for supporting the upcoming versions as the
previous ones.

If you want me to maintain the older version or some improvements, you can
always contract with me.

Best regards

Sylvain

**Comment by wyyang** [comment]
Are there specific JDK 11 code or is it just a compilation configuration? Also, when I tried to compile the code, it failed with error 
```
[ERROR] Failed to execute goal org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:1.18.0.372:sonar-plugin (default-sonar-plugin) on project sonar-slack-notifier: Plugin class not found: 'com.byoskill.sonar.slacknotifier.SlackNotifierPlugin -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.sonarsource.sonar-packaging-maven-plugin:sonar-packaging-maven-plugin:1.18.0.372:sonar-plugin (default-sonar-plugin) on project sonar-slack-notifier: Plugin class not found: 'com.byoskill.sonar.slacknotifier.SlackNotifierPlugin
```
I tried both with latest in master branch, or tagged to 8.0.0 beta. Did I set up my environment wrong? Thanks.
